cloudhub-2/modules/ROOT/pages/ps-assign-permissions.adoc

```diff
@@ -6,12 +6,18 @@ assigned to your user account or a team to which you belong.
 To assign permissions, you need access to Access Management and the Organization Administrator permission.
 
 * To set up and manage a private space, you need the CloudHub Network Administrator permission.
-* To view a private space, you need the CloudHub Network Viewer permission.
-** To view the schedules in applications deployed to CloudHub 2.0 and Runtime Fabric, you must assign the Exchange Viewer permission to the Business Organization that owns the asset.
+* To view a private space as an individual user within an organization, you need at least one of the following permissions:
+** Cloudhub Network Viewer, which enables you to view private spaces and their details.
+** Organization Administrator, which enables you to manage and update private spaces.
+** Cloudhub Network Administrator, which enables you to manage and update private spaces.
++
+Without at least one of these permissions, you can't see any details upon selecting the *Private Spaces* tab. Instead, you receive a *Not Authorized* message.
```

Comment:
Instead of using several negations, try to turn them into positives such as:
If you miss one of these permissions, you see no details when selecting the *Private Spaces* tab. Instead, you see a *Not Authorized* message.